fiasco/tests/test_ion.py

```diff
@@ -160,6 +160,11 @@ def test_radd_ions(ion, another_ion):
     assert collection[1] == ion
     assert collection[0] == another_ion
 
+def test_transitions(ion):
+    trans = ion.transitions
+    assert isinstance(trans, fiasco.Transitions)
+    assert len(trans) == 1
```

Comment:
Why is the expected result 1?

Reply:
🤷 not sure, I think this is just a placeholder for now. The test isn't passing currently, and is requesting user input which pytest doesn't like. I'm guessing it's because there is a file missing somewhere.

Reply:
I'm confused. All the failures I see on the CI are because Fe 5 has 361 transitions and not 1.

Reply:
🤣 I got confused with https://github.com/wtbarnes/fiasco/pull/104

Reply:
Okay, should be fixed now, with added bonus tests!